packages/bsky/src/index.ts

```diff
@@ -96,6 +102,15 @@ export class BskyAppView {
       )
     }
 
+    let revenueCatClient: RevenueCatClient | undefined
+    if (config.revenueCatV1ApiKey && config.revenueCatWebhookAuthorization) {
```

Comment:
Probably not a huge deal, but I'm a bit more inclined to hinge this off of the existence of just one variable (for instance the api key, or even a new "enabled" string) and then throw if other vairables are missing. Basically make it a bit less likely for silent failures

for instance
```ts
if(config.revenueCatV1ApiKey) {
  assert(config.revenueCatWebhookAuthorization)
  // ...
}
```